src/utils.jl

```diff
@@ -1,4 +1,18 @@
-const DEBUG = false
+using Preferences
+
+const DEBUG = @load_preference("use_debug", false)
+
+"""
+toggle_ferrite_debug_mode(use_debug = true)
+
+Helper to turn on or off debug helpers in Ferrite.
+"""
+function toggle_ferrite_debug_mode(use_debug = true)
```

Comment:
Perhaps 
```suggestion
function debug_mode(enable = true)
```
and not exporting it?